FAQ.md

```diff
@@ -209,6 +227,39 @@ If we wanted to only warn for things that throw, then we would warn when calling
  
 NOTE(domenic): there is a conflict between static analysis and dynamic analysis. Consider the example let x : number = true ? 0 : false; A runtime type system would have no problem with this, whereas what you're proposing would disallow it. This is probably worth highlighting at a higher level in the document: it seems like you are leaning toward a statically-checkable model, which there is precedent for for JS but I believe other optionally-typed languages went a different direction.
 
+## Are you breaking the web?
+
+Nope. See [are all existing programs correct?](#are-all-existing-programs-correct).
+
+## Are all existing programs correct?
+
+NOTE(erights): specifically, are they all still statically valid? and if not, how do you avoid #breakingtheweb?
+
+[Interpreters](#terminology) remain unchanged, hence keeping all existing programs correct. For the newly introduced [typechecker](#terminology) some (previously valid) programs may be (desirably) invalidated.
+
+```javascript
+var x = 'foo'; // OK
+var y= x - 1; // OK
+// Typechecker Error, although Interpretation remains semantically and statically valid.
+var z = 1 - true; // Error
+```
+
+## What's the default for untyped programs?
+
+NOTE(erights): does all untyped code default to Any? We found by experience that it is useful sometime to use an unamed Type to be a better default in certain occasions / compilation contexts.
```

Comment:
Yep, i made that clarification. I'll leave that NOTE as a TODO for me to come back to this and eventually understand the trade-offs here.